go.mod

```diff
@@ -3,18 +3,31 @@ module github.com/aquasecurity/harbor-scanner-trivy
 go 1.13
 
 require (
+	github.com/Microsoft/hcsshim v0.8.7 // indirect
```

Comment:
Just curious why do we have so many changes in `go.mod` even if we do not add new modules?

Reply:
Yeah it's intriguing. I even tried to run `go mod tidy` but these changes seem to persist.